### ISSUE TITLE
data producer information

### ISSUE THREAD

**Issue by schellep** [dzhw/metadatamanagement#3106]
I am cleaning up my notes (in this case from a MoGAADat-Interview (I3, Paragraph 74), and found this task for a fare future of the mdm: 

As a data producer I wan’t to be informed what happens to data I produced. 
-	How many students, researchers are using the data in the last year?
-	Which articles have been successfully published?

I really don’t know, if it is possible to give this feedback with an individual report automatically via mdm/dlp? Or if there should be rather a data publisher newsletter (including overall “usage”)?

**Comment by schellep** [comment]
@UteH, ich hab nur im MDM (nicht im DLP) geguckt, ob es dazu schon ein Issue gibt und ich war mir nicht sicher über die ganzen labels, außer das es nur was für den backlog ist ...

**Comment by UteH** [comment]
@schellep definitely valid requests!

@AndyDaniel1 who is responsible for these intersectional requests 🤔 ? I guess team userservice and the data ingest team need to be involved if the issue is tackled?

**Comment by schellep** [comment]
And maybe for the first years (~5y) a (yearly) notification that knowbody used my data 😢 could be a relevant for data provider, too.

**Comment by AndyDaniel1** [comment]
@schellep @UteH this is not an issue directly related to the MDM, as the MDM has no information about user contracts. I will transfer it to the user service repo.

**Comment by UteH** [comment]
@AndyDaniel1 ist das nicht eher datenaufnahmeprozesse?
(userservice ist nur für Datennutzung oder auch Datenaufnahme?)

**Comment by AndyDaniel1** [comment]
@UteH Nein, hier geht es um das reporting der Datennutzung, nicht um die Aufnahme. 
![grafik](https://user-images.githubusercontent.com/17981185/174585120-bd3f0c41-9431-46da-a115-19c860b58931.png)
